Validation and serialization (#2023)
fastify/fastify: docs/Errors.md

A schema with the same `$id` already exists.

<a name="FST_ERR_SCH_VALIDATION_BUILD"></a>
#### FST_ERR_SCH_VALIDATION_BUILD

> The JSON schema provided for validation to one route is not valid.

Reviewer: ```suggestion
The JSON schema provided for validation to a route is not valid.
```

<a name="FST_ERR_SCH_SERIALIZATION_BUILD"></a>
#### FST_ERR_SCH_SERIALIZATION_BUILD

> The JSON schema provided for serialization to one route is not valid.

Reviewer: ```suggestion
The JSON schema provided for serialization of a route response is not valid.
```
